backend/cart-service/deploy/src/main/java/by/jenka/rss/backend/cartservice/config/AwsConfig.java

```diff
@@ -0,0 +1,9 @@
+package by.jenka.rss.backend.cartservice.config;
+
+
+public class AwsConfig {
+
+    public static String DEFAULT_REGION = "eu-north-1";
+    public static String DEFAULT_ACCOUNT = "590184055641";
```

Comment:
IMHO should be taken from .env file and not hard coded.

Reply:
agree